app/helpers.php

```diff
@@ -33,6 +34,18 @@ function edit_image($image, $coords)
 }
 
 /**
+ * Query for total reactions for a post.
+ *
+ * @param int $postableId
+ * @param int $postableType
+ * @return int total count
+ */
+function getTotalPostReactions($postableId, $postableType)
+{
+    return Reaction::where(['postable_id' => $postableId, 'postable_type' => $postableType])->count();
```

Comment:
@DFurnes ah great point! I'm not sure if this works with the polymorphic relation though? When I do 
```
    return Post::withCount(['reactions' => function ($query) {
        $query->where(['postable_id' => $postableId, 'postable_type' => $postableType]);
    }])->get();
```
I get the error
```
{
  "error": {
    "code": 500,
    "message": "Array to string conversion"
  }
}
```
It gets mad in the HasOneOrMany.php file, with `$this->localKey` (line 412) being 
```
array:1 [
  0 => "event_id"
]
``` 
instead of an expected string. I think it's because of the polymorphic relationship after reading more about `$this->localKey` [here](http://stackoverflow.com/questions/26803411/what-is-local-key-in-laravel-eloquent)?